### PR TITLE
Fix displaying the locked banner by fixing date_locked

### PR DESCRIPTION
**Pull request by puiterwijk**

### CODE REVIEW

**Comment by puiterwijk** [comment]
Please read the commit messages for explanations.

**Comment by puiterwijk** [comment]
The reason this was not a problem with 2.0: it did not have the date_locked attribute.

File: bodhi/server/consumers/masher.py [hunk 1]
```diff
@@ -405,23 +403,9 @@ def unlock_updates(self):
         self.log.debug('Unlocking updates')
         for update in self.updates:
             update.locked = False
+            update.locked_date = None
```
Comment (trishnaguha) [inline bodhi/server/consumers/masher.py]: It should be `update.date_locked` instead?

Comment (puiterwijk) [reply bodhi/server/consumers/masher.py]: Correct, fixed that. Thanks.

File: bodhi/server/consumers/masher.py [hunk 2]
```diff
@@ -406,23 +403,9 @@ def unlock_updates(self):
...
-    def locked_date_for_update(self):
-        """ Return the date & time when an update has been locked """
-        for update in self.updates:
-            # Return datetime when an update is locked and datetime is
-            # not yet assigned to the column date_locked
-            if update.locked and not update.date_locked:
-                update.date_locked = datetime.utcnow()
-                self.db.commit()
-                self.db.refresh(update)
-                return update.date_locked
```
Comment (bowlofeggs) [inline bodhi/server/consumers/masher.py:419]: This seems like a pretty strange pattern - to loop over updates until you hit one that's locked without a date, set its date, and… return? What about the rest of the updates in `self.updates`?

**Comment by bowlofeggs** [comment]
@puiterwijk These commits seem like they should be squashed together. The first one shouldn't be applied alone since the second one has the code to clear the date_locked.

Additionally, can you throw a `fixes #952` in your commit message so it automatically ties to the bug? I updated the bug to be about this instead of about the session stuff.

**Review by bowlofeggs — APPROVED** [review]
Thanks so much!

**Comment by bowlofeggs** [comment]
I will write unit tests for this later. For now I am happy to go ahead and merge this since we need it ASAP.

**Pull request merged**